lib/msf/core/exploit/remote/nuuo.rb

```diff
@@ -0,0 +1,196 @@
+require 'msf/core/exploit/tcp'
+
+###
+#
+# This module exposes methods that may be useful to exploits that deal with
+# servers that speak Nuuo NUCM protocol for their devices and management software.
+#
+###
+module Msf
+module Exploit::Remote::Nuuo
+  include Exploit::Remote::Tcp
+
+  #
+  # Creates an instance of an Nuuo exploit module.
+  #
+  def initialize(info = {})
+    super(update_info(info,
+      'Author'         =>
```

Comment:
I see your evil plan there :-)

Reply:
:) I'm not planning to dig deeper (for the moment), but pointing it out to whoever wants to look into it...